Personel/Shubhangi/problem solving/9 sep/bubblesort.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+
+int main()
+{
+
+   int count, temp, i, j, number[30];
+
+   printf("How many numbers are u going to enter?: ");
+   scanf("%d",&count);
+
+   printf("Enter %d numbers: ",count);
+
+   for(i=0;i<count;i++)
+   scanf("%d",&number[i]);
+
+   for(i=count-2;i>=0;i--)
```

Comment:
for(j = 0; j < 5; j++) {
	if((a[j]%2) != 0) {
		total += a[j];
	}	
} for loop and if loop should be like this